charts/citrix-k8s-cpx-ingress-controller/v1.6.1/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: "1.6.1"
+description: A Helm chart for Citrix ADC CPX with Citrix ingress Controller running as sidecar.
+name: citrix-k8s-cpx-ingress-controller
+version: 1.6.1
+icon: https://raw.githubusercontent.com/citrix/citrix-helm-charts/gh-pages/icon.png
```

Comment:
can we please have the icon in the parent directory and refer it as `file://` so users with the air-gapped environment can also view it properly.  please refer to [this](https://github.com/rancher/charts/blob/master/charts/wordpress/v7.3.8/Chart.yaml#L6) as example, thanks.